config/datasets/ot_gcp.yaml

```diff
@@ -1,10 +1,33 @@
 # Release specific configuration:
 release_version: "24.01"
+version: "XX.XX"
 release_folder: gs://genetics_etl_python_playground/releases/${datasets.release_version}
 
 inputs: gs://genetics_etl_python_playground/input
 outputs: gs://genetics_etl_python_playground/output/python_etl/parquet/${datasets.version}
 
+## Datasets:
```

Comment:
I'm not exactly sure how detailed these configs should be. I assume all the files the ETL DAG has to have access to should be here. However this makes a bit tricy to make sure the file parameters are consistent across this config and the DAG definition.

Reply:
unfortunately, YAML config and DAG configuration are at the moment two separate entities that don't talk to each other